tests/testthat/test-shapefile.R

```diff
@@ -25,15 +25,20 @@ test_that("Test get_park_boundaries for file and shapefile object", {
 })
 
 test_that("Default NPS boundary URL is valid", {
-  expect_true(RCurl::url.exists(nps_boundary_url()))
+  expect_false(httr::http_error(nps_boundary_url()))
 })
 
-test_that("Local shapefiles are readable", {
-  aoi <- get_shapefile(path = system.file("shape/nc.shp", package="sf"))
-  expect_s4_class(aoi, "SpatialPolygonsDataFrame")
-})
+# test_that("Local shapefiles are readable", {
```

Comment:
Heya @WilliamsTravis is there a need to comment this out? I'd prefer to either delete it if we don't need it, or keep it if it is testing something important. 

For ease of maintenance, it let's avoid chunks of commented out code. Eventually these can rot and become less relevant, and they are confusing (e.g,. do we need it? do we not need it?).